docs/4.2/trustedclusters.md

```diff
@@ -311,6 +311,36 @@ and set `enabled` to "false", then update it:
 ```bsh
 $ tctl create --force cluster.yaml
 ```
+## Sharing Kubernetes groups between Trusted Clusters
+
+Below is an example of how to share a kubernetes group between trusted clusters. 
+
+In this example, we have a root trusted cluster with a role `main` and kubernetes groups:
+
+```yaml
+kube_groups: ["system:masters"] 
+```
+SSH logins:
+
+```yaml
+logins: ["root"]
+```
+The leaf cluster can choose to map this `main` cluster to it's own. The `remote-admin` cluster in the trusted cluster config:
```

Comment:
let's user 'root' and leaf here.

Reply:
Sounds good, I've updated to reflect this throughout the section